components/hab/src/cli.rs

```diff
@@ -787,7 +787,7 @@ fn sub_pkg_build() -> App<'static, 'static> {
                                               .long("docker"));
     }
 
-    if cfg!(target_os = "windows") {
+    if cfg!(windows) {
```

Comment:
Not exactly your change, but as long as we're touching this: I think this pattern is more suceptible to future defects than it could be. Plus, it's different than the `native_studio_support` code just above it. I suggest we change this from
```rust
    if cfg!(windows) {
        sub.arg(…)
    } else {
        sub
    }
```
to
```rust
    if cfg!(windows) {
        sub = sub.arg(…)
    }

    sub
```

Reply:
Good call; will fix.

Reply:
Fixed in https://github.com/habitat-sh/habitat/pull/6327/commits/52246278888fbba23e5ede6750a0d685f7000722